aws/resource_aws_ec2_capacity_reservation.go

```diff
@@ -205,22 +208,14 @@ func resourceAwsEc2CapacityReservationRead(d *schema.ResourceData, meta interfac
 	d.Set("instance_match_criteria", reservation.InstanceMatchCriteria)
 	d.Set("instance_platform", reservation.InstancePlatform)
 	d.Set("instance_type", reservation.InstanceType)
+	d.Set("owner_id", reservation.OwnerId)
 
 	if err := d.Set("tags", keyvaluetags.Ec2KeyValueTags(reservation.Tags).IgnoreAws().IgnoreConfig(ignoreTagsConfig).Map()); err != nil {
 		return fmt.Errorf("error setting tags: %s", err)
 	}
 
 	d.Set("tenancy", reservation.Tenancy)
-
-	arn := arn.ARN{
-		Partition: meta.(*AWSClient).partition,
-		Service:   "ec2",
-		Region:    meta.(*AWSClient).region,
-		AccountID: meta.(*AWSClient).accountid,
-		Resource:  fmt.Sprintf("capacity-reservation/%s", d.Id()),
-	}.String()
```

Comment:
When removing this type of manual ARN building, we should double check that it is not in the `website/docs/index.html` file `skip_requesting_account_id` list of resources that are documented as potentially not working. 👍